java/ql/lib/semmle/code/java/security/ZipSlipQuery.qll

```diff
@@ -40,5 +40,5 @@ module ZipSlipFlow = TaintTracking::Global<ZipSlipConfig>;
  * A sink that represents a file creation, such as a file write, copy or move operation.
  */
 private class FileCreationSink extends DataFlow::Node {
-  FileCreationSink() { sinkNode(this, "create-file") }
+  FileCreationSink() { sinkNode(this, "path-injection") }
```

Comment:
Similarly here, this includes the old `read-file` sinks now, which changes behavior.